doc/LICENSE.txt

```diff
@@ -9,9 +9,11 @@ All rights reserved.
 Contains code from NumPy, Copyright (c) 2005-2016, NumPy Developers.
 All rights reserved.
 
-Contain CnMeM under the same license with this copyright:
+Contains CnMeM under the same license with this copyright:
 Copyright (c) 2015, NVIDIA CORPORATION. All rights reserved.
 
+Contains frozendict code from slezica’s python-frozendict(https://github.com/slezica/python-frozendict/blob/master/frozendict/__init__.py), Copyright (c) 2012 Santiago Lezica. All rights reserved. 
```

Comment:
Here it is given it's rights reserved, https://github.com/slezica/python-frozendict/blob/master/LICENSE.txt . I'm very new to this. I thought since it was given there, i should include it as well